src/systems/actions.cairo

```diff
@@ -82,11 +81,18 @@ pub mod actions {
 
             // Write the new moves to the world.
             world.write_model(@moves);
-          
+
             // Emit an event to the world to notify about the player's move.
             world.emit_event(@Moved { player, direction });
         }
     }
+
+    #[generate_trait]
+    impl InternalImpl of InternalTrait {
+        fn world_default(self: @ContractState) -> dojo::world::WorldStorage {
+            self.world(@"dojo_starter")
+        }
+    }
```

Comment:
Good to have put this down here, could we add a comment on why this is interesting practice to have? Since the namespace is the same in this example, it's a good way for the users to not hardcode it. 👍